schematic/schemas/commands.py

```diff
@@ -114,35 +126,44 @@ def convert(
                 for warning in war:
                     logger.warning(warning)
 
+    if output_path:
+        output_jsonld = output_path
+
+    if output_jsonld is None:
+        output_file_no_ext = re.sub("[.](jsonld|csv|pickle)$", "", schema)
+    else:
+        output_file_no_ext = re.sub("[.](jsonld|csv|pickle)$", "", output_jsonld)
+
+    logger.info(
+        "By default, the JSON-LD output will be stored alongside the first "
+        f"input CSV or JSON-LD file. In this case, it will appear here: '{output_jsonld}'. "
+        "You can use the `--output_jsonld` argument to specify another file path."
+    )
+
+    if output_type in ["graph", "all"]:
+        logger.info("Export graph to pickle.")
```

Comment:
Could you update this to contain more context in the log message. Think about what would be useful to know when this graph was exported to pickle. Such as:

1. The output location
2. The name of the file
3. The type of output